src/analytics.js

```diff
@@ -1,13 +1,24 @@
 const Countly = require('countly-sdk-nodejs')
 const { ipcMain } = require('electron')
 const { COUNTLY_KEY } = require('./common/consts')
+const { join } = require('path')
+const { app } = require('electron')
+const { existsSync, mkdirSync } = require('fs')
 
 module.exports = async function (ctx) {
+  // workaround: recursive mkdir https://github.com/Countly/countly-sdk-nodejs/pull/14
+  const storage_path = join(app.getPath('home'), './.ipfs-desktop/countly-data')
```

Comment:
Afraid this won't work nicely on non-linux platforms.

Please define `const` at the top and change `storage_path` to [`app.getPath('appData')/countly-data`](https://www.electronjs.org/docs/api/app#appgetpathname)

Reply:
Actually, we want `userData`, which will automatically include the suffix with IPFS Desktop name :+1: